hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -127,13 +111,15 @@ private void createSymbolicLink() {
           log.info("Unable to unlink old sym link. Link may not exist. Exit code: " + exitCode);
         }
       } catch (IOException e) {
-        log.fatal("Could not unlink " + hdfsBinaryPath + ": " + e);
-        System.exit(1);
+        log.fatal("Could not unlink " + hdfsBinaryPath, e);
+        throw e;
```

Comment:
This is a serious error and we need to actually shutdown the executor. We need the System.exit(1) here instead of just throwing an exception (not a runtime exception).